<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManaUserItem</class>
 <widget class="QWidget" name="ManaUserItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ManaUserItem</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ManaUserItem{background-color: rgb(255, 255, 0);}
QLabel{
font: 16pt &quot;Adobe 黑体 Std R&quot;;}

QLabel,QToolButton{
background-color: rgba(255, 255, 255, 0);}</string>
  </property>
  <widget class="HeadPor" name="frameHead">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>958</x>
     <y>40</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../BMS.qrc">
     <normaloff>:/用户界面小按钮/Resources/小按钮/未选中.png</normaloff>:/用户界面小按钮/Resources/小按钮/未选中.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="popupMode">
    <enum>QToolButton::DelayedPopup</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="ID">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="power">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>70</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>437</x>
     <y>20</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sex">
   <property name="geometry">
    <rect>
     <x>463</x>
     <y>70</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="institute">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>20</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="className">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>70</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>117</y>
     <width>1100</width>
     <height>3</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>HeadPor</class>
   <extends>QFrame</extends>
   <header location="global">headpor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../BMS.qrc"/>
 </resources>
 <connections/>
</ui>
